examples/asr/librispeech_emformer_rnnt/global_stats.py

```diff
@@ -0,0 +1,69 @@
+import json
+import logging
+import pathlib
+from argparse import ArgumentParser
+
+import torch
+import torchaudio
+from utils import GAIN, piecewise_linear_log, spectrogram_transform
+
+logger = logging.getLogger()
+
+
+def generate_statistics(samples):
+    E_x = 0
+    E_x_2 = 0
+    N = 0
+
+    for idx, sample in enumerate(samples):
+        mel_spec = spectrogram_transform(sample[0].squeeze()).transpose(1, 0)
+        scaled_mel_spec = piecewise_linear_log(mel_spec * GAIN)
+        sum = scaled_mel_spec.sum(0)
+        sq_sum = scaled_mel_spec.pow(2).sum(0)
+        M = scaled_mel_spec.size(0)
+
+        E_x = E_x * (N / (N + M)) + sum / (N + M)
+        E_x_2 = E_x_2 * (N / (N + M)) + sq_sum / (N + M)
+        N += M
+
+        if idx % 100 == 0:
+            logger.info(f"Processed {idx}")
+
+    return E_x, (E_x_2 - E_x ** 2) ** 0.5
+
+
+def cli_main():
+    parser = ArgumentParser()
```

Comment:
Can you add general CLI `description`?
My recommendation is to add module level docstring and reuse it in `description` argument. https://github.com/pytorch/audio/blob/main/examples/libtorchaudio/speech_recognition/parse_librispeech.py